sections/conclusion/discussion.tex

```diff
@@ -0,0 +1,101 @@
+
+\subsection{Comparing the final product to the MVP}
+In \autoref{sec:mvp} an MVP was defined for a viable game solution, split into functional and non-functional requirements.
+
+\subsubsection{Functional requirements}
+The functional requirements defined the core features of the game.
+The first three requirements are defined as starting the game, connecting clients and server and users viewing their position.
+All of this is accomplished through the network protocol defined in \autoref{app:network}, specifically TCP message 3 for game start and UDP message 0 to update player positions.
+Connecting players is done through the lobby screen based on IP.
+The next few requirements deal with actually playing the game, requiring the player to be able to view a playing field, goals, the players, the ball and current score, as well as being able to win a game based on their current score.
+This has all been accomplished in the Unity game, as shown in REFREFREFREF.
+\todo{tænker vi kan ref til sprint-5 coinclusion til at vise det. Der skal vel være noget afsluttende ift spillet}
+Finally, the game should be dynamic, allowing for an arbitrary number of players and goals to win as well as different sizes of playing fields.
+This is partly accomplished.
+The game supports an arbitrary number of goals to win, however, in terms of players there is currently a maximum capacity of four players in a single game.
+This is communicated by the host to the clients through TCP message 6 in \autoref{app:network}.
+The playing field scales based on the user input, meaning that requirement is supported, but fields have to be shaped like a rectangle or a square in order to support the current implementation of generating new goal zones.
+
+\subsubsection{Non-functional requirements}
+The non-functional requirements to be evaluated through a usability test.
+A usability test was conducted in \todo{ref til 18 maj}, but it could not be conducted properly due to the circumstances.
+However, the non-functional requirements state that the game should be pleasant to look at and not cause discomfort for players, and there should be enough updates to represent the real positions of the players.
+None of the testers experienced nausea or discomfort during the playing of the game, indicating that the first requirement was likely achieved.
+There were only six testers however, which is likely not enough testers to determine this completely.
+The testers could have been outliers, or there could be possible outliers who would find the playing of the game uncomfortable.
+As explained in \todo{18 maj ref}, there were some slight issues in terms of updates when playing the game outside.
+Occasionally, updates would stop being sent for a period of several seconds.
+This caused play to be interrupted.
+\todo{Den her sætning herunder. Er det godt nok sådan her? Eller er det at vi blamer tags?}
```

Comment:
Altså vi der kunne måske godt spekuleres i forskellige årsager til det her men nej vi ved jo ikke hvad der er skyld i det

Reply:
Der kan evt. tilføjes noget med at vi har forsøgt at gøre op for den lave update rate ved implementering af lerping når det engang er merged ind